x-pack/plugins/apm/server/lib/transaction_groups/root_cause_analysis/get_duration_limit.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { ESFilter } from '../../../../typings/elasticsearch';
+import { TRANSACTION_DURATION } from '../../../../common/elasticsearch_fieldnames';
+import { ProcessorEvent } from '../../../../common/processor_event';
+import {
+  Setup,
+  SetupTimeRange,
+  SetupUIFilters,
+} from '../../helpers/setup_request';
+
+export async function getDurationLimit({
+  durationPercentile,
+  backgroundFilters,
+  setup,
+}: {
+  durationPercentile: number;
+  backgroundFilters: ESFilter[];
+  setup: Setup & SetupTimeRange & SetupUIFilters;
```

Comment:
Are we passing in `setup` just to align with other endpoints? Looks like we're only using `search` here.

Reply:
Yes, just for alignment. We might need more things from `setup` in the future.